tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -59,6 +61,61 @@ def test_pcluster_configure(
     clusters_factory(config_path)
 
 
+@pytest.mark.regions(["us-east-1"])
```

Comment:
I'd suggest no choosing us-east-1 or eu-west-1 as test region due to integration test region balance, unless you need these regions for particular purpose.

Reply:
I think logic below might be specific to us-east-1, and I think we should only run this test on 1 dimension combination.
It would be ok to keep things as is, but general it is good to avoid "popular" regions in order to balance the tests. Or even better, make the logic generic so we could run on other regions as well.